### DISCUSSION TITLE
Is it possible to use panda AND tailwind together with park?

### DISCUSSION THREAD

**Discussion by DawidWraga** [cschroeter/park-ui#63] [Q&A]
Is it possible to use the tailwind and panda version of park-ui together? 

this would be really helpful as I prefer panda but sometimes tailwind is more convenient eg compatability with tremor.so or using tailwind templates.

**Comment by cschroeter** [comment]
I'm pretty sure it can be achieved. The only thing you would have to watch out for is the layers in Panda and Tailwind.

Panda has a [minimal setup](https://panda-css.com/docs/guides/minimal-setup) guide that would be very helpful in that case.
